hyper/cli.py

```diff
@@ -160,13 +161,16 @@ def set_request_data(args):
         elif i.sep == SEP_QUERY:
             params[i.key] = i.value
         elif i.sep == SEP_DATA:
-            body[i.key] = i.value
+            value = i.value
+            if is_py2:  # pragma: no cover
+                value = value.decode(PREFERRED_ENCODING)
+            body[i.key] = value
 
     if params:
         args.url.path += '?' + urlencode(params)
 
     if body:
-        content_type = 'application/json; charset=%s' % FILESYSTEM_ENCODING
+        content_type = 'application/json; charset=utf-8'
```

Comment:
Thank you for the references. I learned a lot. Originally, I read httpie's client and I tried to tell that, but it has already been fixed. :smile: 
https://github.com/jakubroztocil/httpie/commit/6e7e2f2eeaf5d07060303735d8f183e526447f25